docker-compose.yml

```diff
@@ -16,7 +16,7 @@ services:
       - "4151:4151"
   nsqadmin:
     image: nsqio/nsq
-    command: /nsqadmin --lookupd-http-address=nsqlookupd:4161
+    command: /nsqadmin --lookupd-http-address=nsqlookupd:4171
```

Comment:
conflicting with something locally?

Reply:
oh nm. this is totally unnecessary and would break nsqadmin. I can't even docker.